recipes/pdfcombine/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pdfcombine" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "23e3af619a4859008745ac61a6040a230023cd4e5c4c667d9b8ea248726f47ed" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/tdegeus/pdfcombine/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+
+requirements:
+  host:
+    - python
+  run:
+    - python
+    - docopt
+    - click
+    - ghostscript
+
+test:
+  commands:
+    - test -f ${PREFIX}/bin/pdfcombine  # [unix]
```

Comment:
Since `pdfcombine` is an executable, the test command should be `pdfcombine --help` instead of checking if the file exists.

Reply:
Better indeed!

Reply:
@npavlovikj How do I actually execute a Python script in Windows ☺️?

The following does not work:
```none
pdfcombine --help
python pdfcombine --help 
```

Reply:
I fact, how does Windows know to use Python. Do I have to name the command `pdfcombine.py`?

Reply:
You actually use the script the same way as you do on Unix - you don't need the `.py` extension nor `python` prefix.